src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/FeatureStrategyItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTheme } from '@mui/material';
+import { PlaygroundResultChip } from '../../../../PlaygroundResultChip/PlaygroundResultChip';
+import {
+    PlaygroundStrategySchema,
+    PlaygroundRequestSchema,
+} from 'hooks/api/actions/usePlayground/playground.model';
+import { StrategyExecution } from './StrategyExecution/StrategyExecution';
+import { useStyles } from './FeatureStrategyItem.styles';
+import { StrategyItemContainer } from 'component/common/StrategyItemContainer/StrategyItemContainer';
+import { objectId } from 'utils/objectId';
+
+interface IFeatureStrategyItemProps {
+    strategy: PlaygroundStrategySchema;
+    index: number;
+    input?: PlaygroundRequestSchema;
+}
+
+export const FeatureStrategyItem = ({
+    strategy,
+    input,
+    index,
+}: IFeatureStrategyItemProps) => {
+    const { result } = strategy;
+    const { classes: styles } = useStyles();
+    const theme = useTheme();
+    const label =
+        result.evaluationStatus === 'incomplete'
```

Comment:
Question: if the evaluationStatus is incomplete and result is false, that means we know for sure that the strategy is false, even if we can't evaluate the actual strategy (in other words: there are failing constraints). How do we show that?

In other words, even if we can't evaluate a strategy, its constraints can tell us that it's false. In that case, should it show as false?

Reply:
Each constraint shows visually that it failed.  The strategy will still show us Unevaluated